demos/manifest.json

```diff
@@ -49,5 +49,14 @@
 	"dependencies": {
             "cert-manager": true
         }
+    },
+    "px-mongo": {
+        "description": "Microservice demo that generates traffic between the Mern-K8s demo app and MongoDB.",
+        "instructions": [
+            "Load testing has been automatically launched for px-mongo",
+            "",
+            " Use the px/mongodb_data and px/http_data scripts to view the backend API and MongoDB traffic",
+            " flowing through the demo app."
```

Comment:
I think we should mention that an end user's vizier must be `v0.14.8` or later. `v0.14.7` is the latest version, so we need to release that this / next week.

Otherwise they won't have the new mongo tracing functionality.